.github/scripts/tar-files.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Installs all sources relevant to the rootfs build as tar files
+
+# Create the tar-files directory
+mkdir tar-files
+cd tar-files
+
+# Any single arguement ensures that only the linux source files will be downloaded
+if [[ -z $1 ]]; # If no argument is given
+     then
```

Comment:
Usual comment about lonely `then`.

Reply:
Thank you for taking a look at this, all useful comments that I will keep in mind for future scripting.

Regarding the merge, if you click the little arrow next to the 'Merge pull request' button the plan is to use the 'squash and merge' option to condense this into a single commit.